integration/lsp/viewengine_spec.ts

```diff
@@ -124,6 +124,7 @@ describe('initialization', () => {
         definitionProvider: true,
         typeDefinitionProvider: true,
         hoverProvider: true,
+        referencesProvider: true,
```

Comment:
Actually, now that we have the `ivy` flag, we should enable this only for Ivy

Reply:
Good call. Done